accumulator/pollardutil.go

```diff
@@ -71,6 +71,15 @@ func (n *polNode) prune() {
 	}
 }
 
+// getSize returns the size of the polNode and all the nieces below it in bytes.
+func getSize(n *polNode) int64 {
+	if n == nil {
+		return 0
+	}
+	// hash (32), two pointers (16), and a bool (1)
+	return (getSize(n.niece[0]) + 32 + 16 + 1 + getSize(n.niece[1]))
```

Comment:
Using `unsafe.Sizeof` might be better here.
https://play.golang.org/p/fY3YmirNFSe
The Go compiler aligns the bool to 8 bytes.

Reply:
Fixed.